### ISSUE TITLE
Support multiple urls in searchurls similar to quickmark

### ISSUE THREAD

**Issue by user471** [tridactyl/tridactyl#1966]
It would be very helpful if It would be possible to use one keyword to search in multiple engines
```
set searchurls.video https://www.youtube.com/results?search_query=%s https://vimeo.com/search?q=%s
```
`:tabopen video foo` would open two tabs

**Comment by bovine3dom** [comment]
If you have video1 and video2 searchurls,

`com vidopen jsb -p tri.excmds.tabopen("video1",JS_ARG); tri.excmds.tabopen("video2",JS_ARG)`

And then `vidopen how to quit emacs` might work.

**Comment by user471** [comment]
@bovine3dom 
Thanks. When I use `vidopen how to quit emacs` only last work `emacs` is used in search.
How to access all words through `JS_ARG`?

**Comment by bovine3dom** [comment]
Yeah, something odd is going on with the way `command` works with `JS_ARG`. `composite echo how to quit emacs | jsb -p tri.excmds.tabopen("video1",JS_ARG).then(tri.excmds.tabopen("video2",JS_ARG))` works fine but isn't very ergonomic.

I'll try to look into this again later.
